internal/accounts/create.go

```diff
@@ -326,6 +427,38 @@ func getAccountCreatedAddressWithPubKey(
 	return address, nil
 }
 
+// getAccountCreatedWithAPI gets the events from the transaction ID generated from the account creation API. It
+// extracts the newly created address from the account creation event payload.
+func getAccountCreatedWithAPI(
+	service *services.Services,
+	pubKey crypto.PublicKey,
+	txID string,
+) (*flow.Address, error) {
+	result, err := retry(txID, 10, time.Second, service.Transactions.GetTransactionResultByID)
```

Comment:
Need retry because it takes time for the txID to show up on the chain. From my testing, I find that it takes around 3 retries with 2 seconds stop in between to find the transaction. Using 10 retries just to be safe.